config/routes.rb

```diff
@@ -57,21 +57,6 @@
     end
   end
 
-  # to be removed and below used when migrating to contentful
-  static_regexp = %r{
-    accessibility-statement|
-    cookie-policy|
-    new-registration|
-    other-problems-signing-in|
-    privacy-policy|
-    terms-and-conditions|
-    sitemap|
-    settings/cookies|
-    whats-new|
-    wifi-and-data
-  }x
-  get '/:id', to: 'static#show', id: static_regexp, as: :static
-
   constraints !Rails.application.cms? do
     scope module: 'contentful' do
       resources :static, only: %i[show], as: :static_pages, path: ''
```

Comment:
@ebrett what would you like to see with names and namespaces? Remove the `Contentful` module? Do you prefer the new path helper?

Reply:
Contentful is an implementation detail.  Static pages are at the root level of the application, so keep it where it was.

Reply:
I've made this change